workflow/config/default_config.yaml

```diff
@@ -1,6 +1,6 @@
 Docker:
     TPS: True
-    Cytoscape: False
+    Cytoscape: True

```

Comment:
We may want to add comments to the Cytoscape section below that some of these values are ignored when Cytoscape runs in Docker.